sentry/sentry-symfony/4.0/config/packages/sentry.yaml

```diff
@@ -2,10 +2,19 @@ when@prod:
     sentry:
         dsn: '%env(SENTRY_DSN)%'
 
-#        If you are using Monolog, you also need these additional configuration and services to log the errors correctly:
+#        If you are using Monolog, you also need this additional configuration to log the errors correctly:
 #        https://docs.sentry.io/platforms/php/guides/symfony/#monolog-integration
 #        register_error_listener: false
 
+#    monolog:
+#        handlers:
+#            sentry:
+#                type: sentry
+#                level: !php/const Monolog\Logger::ERROR
+#                hub_id: Sentry\State\HubInterface
+
+#    If you are using MonologBundle prior to v3.7, you need to configure the handler as a service instead:
```

Comment:
what about making the recipe conflict with <3.7 and remove the rest?

Reply:
I'm fine either way, don't know whose call this is :smile:

Reply:
I just did :)

Reply:
Fair enough, I've added the conflict on v4, should I do the same for v3?

Reply:
no need to, let's be future-oriented

Reply:
Done